projects/packages/lazy-images/tests/php/test_class.lazy-images.php

```diff
@@ -600,4 +601,22 @@ public function add_skip_lazy_class_to_attributes( $attributes ) {
 		$attributes['class'] .= ' skip-lazy';
 		return $attributes;
 	}
+
+	/**
+	 * Confirms that the lazy images module is not loaded with Gutenberg 16.6.
+	 */
+	public function test_lazy_images_not_loaded_with_gutenberg_16_6() {
+		// Verify without the constant set that the instance is returned.
+		$this->assertInstanceOf( 'Automattic\\Jetpack\\Jetpack_Lazy_Images', Jetpack_Lazy_Images::instance() );
```

Comment:
Just a pet peeve of mine, I find doing it like this is clearer
```suggestion
		$this->assertInstanceOf( Jetpack_Lazy_Images::class, Jetpack_Lazy_Images::instance() );
```
Same below

Reply:
That's fine by me. Pushing a commit now.